src/ssz/type/bit_vector.zig

```diff
@@ -25,6 +25,14 @@ pub fn BitVector(comptime _length: comptime_int) type {
             return bv;
         }
 
+        pub fn toBoolArray(self: *const @This()) [length]bool {
```

Comment:
this could be `out` parameter too even the copy cost is tiny
this makes it conform to other apis, like `hashTreeRoot`